include/particles/particle.tcc

```diff
@@ -500,6 +500,25 @@ void mpm::Particle<Tdim>::map_mass_momentum_to_nodes() noexcept {
   }
 }
 
+//! Map nodal properties to nodes
+template <unsigned Tdim>
+void mpm::Particle<Tdim>::map_properties_to_nodes() noexcept {
+  // Check if particle mass is set
+  assert(mass_ != std::numeric_limits<double>::max());
+
+  // Unit 1x1 Eigen matrix to be used with scalar quantities
+  Eigen::Matrix<double, 1, 1> unit_scalar;
```

Comment:
Why not just a double?

Reply:
I mentioned the reason in the comment made to the line where I use `unit_scalar` (below)